pkg/reloader/reloader.go

```diff
@@ -220,10 +220,10 @@ func (r *Reloader) Watch(ctx context.Context) error {
 
 	r.watches.Set(float64(len(watchables)))
 	level.Info(r.logger).Log(
-		"msg", "started watching config file and recursively rule dirs for changes",
+		"msg", "started watching config file and recursively dirs for changes",
```

Comment:
nit: 
```suggestion
		"msg", "started watching config file and directories recursively for changes",
```

Reply:
Done. I've removed `recursively` because the reloader doesn't watch directories recursively.